src/martok/Martok.ts

```diff
@@ -173,6 +209,10 @@ export class Martok {
     let relativePath = path.resolve(path.dirname(file.fileName));
     if (relativePath.startsWith(this.config.sourceRoot)) {
       relativePath = relativePath.slice(this.config.sourceRoot.length);
+    } else {
+      throw new Error(
```

Comment:
Technically this error is not applicable to types marked `@expand` right?

Reply:
If something is expanded and the type is no longer used then this error won't be thrown. In some cases you can call @ignore on the import statement for something external, as long as you expand the type so that it's not used.